tests/perftest/TollFreeBridge.cs

```diff
@@ -0,0 +1,63 @@
+using System;
+
+using CoreFoundation;
+using Foundation;
+using ObjCRuntime;
+
+using BenchmarkDotNet.Attributes;
+
+using Bindings.Test;
+
+namespace PerfTest {
+
+	[Register ("StringClass")]
+	class StringClass : NSObject {
+			public override string Description => "constant";
```

Comment:
Also it might be interesting to test other strings than "constant", especially ones containing non-ascii characters.

Reply:
Good point. Maybe I should re-write the test, like you suggested, to make that easier...

Reply:
Due to the high probably of conflicts (with next week WWDC) I'll merge this without additional tests. I'll cover that in the next (related) PR